bigquery/setup.py

```diff
@@ -32,8 +32,8 @@
     'enum34; python_version < "3.4"',
     "google-auth >= 1.9.0, < 2.0dev",
     "google-api-core >= 1.15.0, < 2.0dev",
-    "google-cloud-core >= 1.0.3, < 2.0dev",
-    "google-resumable-media >= 0.3.1, != 0.4.0, < 0.6.0dev",
+    "google-cloud-core >= 1.1.0, < 2.0dev",
+    "google-resumable-media >= 0.5.0, < 0.6dev",
```

Comment:
Why is this version bump needed? Looking at the [release notes](https://github.com/googleapis/google-resumable-media-python/releases), there have been no bugfixes since `0.3.1`, and the only new feature is raw download classes (released in `0.5.0`).

Not that I'm opposing it, just asking as a sanity check.

(besides, `google-resumable-media` version pin will have to be updated soon anyway when #10005 is implemented)

Reply:
@plamut Not any specific reason, for the raw download feature and  found in [setup.py](https://github.com/googleapis/google-cloud-python/blob/master/storage/setup.py#L34) file of storage so update it,so if you recommend i will revert back to` 0.3.1`.

Reply:
In theory not much has changed. And you could likely only take core (I am pretty sure given a recent change that is required). The grouping of resumable-media was to keep storage and bigquery in sync with what they require to hopefully avoid odd incompatibilities.

Reply:
Since it appears safe, let's keep the resumable-media version bump then for the reason Chris mentioned.